blockchain/blockchain-wallet-v4-frontend
# Feat/airdrop push

## Description
2201, 2179
Display claim airdrop messaging for tier 2 users and after tier 2 verification

## Change Type
Please enter one or more of the following: 
- Feature

## Testing Steps
1. create tier 2 users
2. see message after completion of tier 2
3. or see message on login

## Code Checklist
